test/lib/adapters/FileSystem_write.js

```diff
@@ -1,17 +1,28 @@
 import path from "node:path";
+import {readFileSync} from "node:fs";
 import {access as fsAccess, constants as fsConstants, mkdir} from "node:fs/promises";
 import {fileURLToPath} from "node:url";
 import test from "ava";
 import {rimraf} from "rimraf";
-import chai from "chai";
-import chaifs from "chai-fs";
-chai.use(chaifs);
-const assert = chai.assert;
-
 import sinon from "sinon";
 
 import {createAdapter, createResource} from "../../../lib/resourceFactory.js";
 
+function getFileContent(path) {
+	return readFileSync(path, "utf8");
```

Comment:
If it's not too much trouble I would prefer to use the async API via [`node:fs/promises`](https://nodejs.org/api/fs.html#promises-api). Reading files non-blocking might even have an impact on the execution time of the tests

Reply:
Good point. I changed it to use the [`node:fs/promises`](https://nodejs.org/api/fs.html#promises-api). 
We could use Promise.all for reading actual and expected file in parallel. For simplicity I did not use that but I don't mind. Feel free to propose your preferred option.

Reply:
Fine with me!